kits/arm/arm.py

```diff
@@ -56,6 +117,13 @@ def __init__(self, params, lookup = None):
             print('Could not load HRDF: {0}'.format(e))
             sys.exit()
         
+        # Create Gripper, if exists
+        if params.hasGripper:
+            self.gripper = Gripper(params.family, params.gripperName)
```

Comment:
There is no checking for failure here, which means if the module wasn't found on the network, you will have a crash later on (in the gripper.update call).  Ideally, you would either error or rectify the state ASAP.  So, either (1) print/throw error here, so effectively the "arm" init function fails if you can't find a gripper, or (2) print and error and set "self.gripper = None" here, so that you don't have a crash later.

Reply:
This network issue is handled in the gripper class - if the gripper doesn't exist on network it'll print an error and shut down. (It wasn't doing it correctly earlier but I've fixed it to quit properly now).

- Is that ok? 
- Is there any other reason the gripper init might fail, and so it would be worth checking at this line as well?